proto/rill/admin/v1/api.proto

```diff
@@ -1635,6 +1636,7 @@ message ListProjectMemberUsergroupsRequest {
   string organization = 1 [(validate.rules).string.min_len = 1];
   string project = 2 [(validate.rules).string.min_len = 1];
   string role = 5; // Optionally filter by role
+  string email = 6; // Optionally filter by user email
```

Comment:
Would this return the usergroups in a project that a user is part of? I thought the requirement was to list the projects in the org that the user had access to. Not sure though